chcemvediet/apps/obligees/tests/test_forms.py

```diff
@@ -115,102 +121,106 @@ def test_submitted_with_nonexisting_obligee_name(self):
-        # Invalid value
-        with self.assertNumQueries(1):
-            with self.assertRaises(ValidationError):
-                field.clean(u'invalid')
-        with self.assertNumQueries(0):
-            with self.assertRaises(ValidationError):
-                field.clean(u'invalid')
-
 class ObligeeFieldWithObligeeWidgetWidget(ObligeesTestCaseMixin, TestCase):
     u"""
     Tests ``ObligeeField`` with ``ObligeeWidget`` widget.
     """
 
     class Form(forms.Form):
-        obligee = ObligeeField(
-                widget=ObligeeWidget(),
-                )
+        obligee = ObligeeField()
 
     class FormWithWidgetAttrs(forms.Form):
         obligee = ObligeeField(
-                widget=ObligeeWidget(attrs={
+                widget=ObligeeWidget(input_attrs={
                     u'class': u'custom-class',
                     u'custom-attribute': u'value',
                     }),
                 )
 
 
-    @unittest.skip(u'FIXME')
     def test_new_form(self):
         form = self.Form()
         rendered = self._render(u'{{ form }}', form=form)
+        lorem = squeeze(render_to_string(u'obligees/texts/widget_no_email.txt'))
         self.assertInHTML(u'<label for="id_obligee">Obligee:</label>', rendered)
         self.assertInHTML(u"""
-                <input class="autocomplete" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text">
+                <input class="form-control pln-autocomplete" data-autocomplete-url="{url}" name="obligee" data-name="obligee" type="text" value="">
                 """.format(url=reverse(u'obligees:autocomplete')), rendered)
         self.assertInHTML(u"""
-                <div class="obligee_widget_details obligee_widget_hide">
-                  <span class="obligee_widget_street"></span><br>
-                  <span class="obligee_widget_zip"></span> <span class="obligee_widget_city"></span><br>
-                  E-mail: <span class="obligee_widget_email"></span>
+                <div class="chv-obligee-widget-details chv-obligee-widget-hide">
+                  <span class="chv-obligee-widget-street"></span><br>
+                  <span class="chv-obligee-widget-zip"></span> <span class="chv-obligee-widget-city"></span><br>
+                  {}: <span class="chv-obligee-widget-email"></span>
+                  <span class="chv-obligee-widget-no-email">
+                  {}
+                    <span class="pln-with-tooltip" data-toggle="tooltip" data-placement="right" title="{}">
+                      <span class="chv-icon-stack chv-icon-lg">
+                        <i class="chv-icon icon-circle chv-color-blue-ll"></i>
+                        <i class="chv-icon chv-icon-inv chv-icon-half icon-help"></i>
+                      </span>
+                    </span>
+                  </span>
                 </div>
-                """, rendered)
+                """.format(_(u'obligees:obligee_widget:email'), _(u'obligees:obligee_widget:no_email'), lorem), rendered)
```

Comment:
V predchadzajucom PR som neprekladal frazy `{% trans ... %}`, tu uz vsak prekladove stringy existuju resp. slovensky a anglicky preklad je rozny, a testujeme tiez error hlasky, ci sa zobrazia v spravnom jazyku pre uzivatela.

Reply:
@viliambalaz Idealne spravit test tak, aby stringy nebolo treba prekladat. Teda, ze budeme pozerat iba na elementy a ich podstatne atributy. Nie vsak uz na konkretne znenia textov.

Obcas sa vsak vyskytnu testy, kde bude treba overit konkretny text. Tak ano, potom bude treba tie texty prekladat tak, ako si to tu spravil.